doc/2-interface/README.md

```diff
@@ -13,6 +13,7 @@ primary topics:
 - [play/edit controls](play-edit-controls.md)
 - [instrument/wavetable/sample list](asset-list.md)
 - [song information](song-info.md)
+- [effect list](effect-list.md)
```

Comment:
Hmmm... not sure whether this could be mistaken for the actual effect list in docs...

Reply:
Perhaps call it "effect list window"?

Reply:
Hmmm... yeah? That could work.

Reply:
Done!